extensions/in-memory/negotiation-store-memory/build.gradle.kts

```diff
@@ -12,15 +12,15 @@
  *
  */
 
+val openTelemetryVersion: String by project
+
 plugins {
     `java-library`
 }
 
-
 dependencies {
     api(project(":spi"))
-
-
+    api("io.opentelemetry:opentelemetry-extension-annotations:${openTelemetryVersion}")
```

Comment:
this is not needed.

Reply:
You are right, thanks! This dependency will be needed if we introduce custom spans later on.